services/language_translator/v3.js

```diff
@@ -22,13 +22,16 @@ module.exports = function (RED) {
   // Otherwise, once set username would never get reset, resulting in a frustrated
   // user who, when he errenously enters bad credentials, can't figure out why
   // the edited ones are not being taken.
-  const SERVICE_IDENTIFIER = 'language-translator';
+  const SERVICE_IDENTIFIER = 'language-translator',
+    LanguageTranslatorV3 = require('ibm-watson/language-translator/v3'),
+    { IamAuthenticator } = require('ibm-watson/auth');
+
   var pkg = require('../../package.json'),
-    LanguageTranslatorV3 = require('watson-developer-cloud/language-translator/v3'),
     //cfenv = require('cfenv'),
     payloadutils = require('../../utilities/payload-utils'),
     serviceutils = require('../../utilities/service-utils'),
     translatorutils = require('./translator-utils'),
+    responseutils = require('../../utilities/response-utils'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4804371)